### DISCUSSION TITLE
Enlarged player for featured item, on index page

### DISCUSSION THREAD

**Discussion by gam-ragnar** [mediacms-io/mediacms#622] [Feature Requests]
Hi all,
Setting the option 
`
VIDEO_PLAYER_FEATURED_VIDEO_ON_INDEX_PAGE = True
`
fails to produce an enlarge featured item on index page

Thanks

**Comment by mgogoulos** [comment]
This option is not implemented on the main branch, and has to be implemented. I'll let this open as enhancement, thanks for opening
